app.py

```
@@ -15,25 +15,30 @@
 
 # 발음 평가
 evaluator = PronunciationEvaluator(subscription_key, service_region)
+
 @app.post("/evaluate-pronunciation/")
-async def evaluate_pronunciation(file: UploadFile = File(...)):
+async def evaluate_pronunciation(reference_text: str, file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)